tests/ui/issues/issue-17546.stderr

```diff
@@ -6,7 +6,6 @@ LL |     fn new() -> NoResult<MyEnum, String> {
   --> $SRC_DIR/core/src/result.rs:LL:COL
    |
    = note: similarly named enum `Result` defined here
-   |
```

Comment:
Remark: hm, I wonder if the spacing originally was intended to help make the separation between the a potential note vs the subsequent help clearer, but IMO this is not very different, so it's fine.